docs.html

```diff
@@ -66,8 +66,14 @@
 <h2>
 <a name="publishing" class="anchor" href="#publishing"><span class="octicon octicon-link"></span></a>Publishing a module</h2>
 
-<p>Push a signed tag to the module then ping an admin (@daenney, @igalic, @nibalizer) and they will publish the module.</p>
-<p>TODO: document module publish process for admins</p>
+<p>Forge publishing is handled by travis and puppet-blacksmith.</p>
+<p>To enable a module to be pushed, cargo-cult the 'deploy' block from puppetcommunity-extlib travis.yml.</p>
+<p>The 'secure' line in the travis.yml is created by:</p>
+<pre>travis encrypt -r puppet-community/puppetcommunity-extlib $the_forge_password</pre>
```

Comment:
where do people get the password from?

<insert link to gpg secured password>
<add info that, if you're not on the list of people entrusted with the password, ask one of them>